pkg/imexport/export.go

```diff
@@ -50,8 +54,95 @@ func createDir(name string, tw *tar.Writer, dir *vfs.DirDoc) error {
 	return err
 }
 
-func export(tw *tar.Writer, fs vfs.VFS) error {
-	root := "/Documents"
+func metadata(tw *tar.Writer, fs vfs.VFS, domain string) error {
+	db := couchdb.SimpleDatabasePrefix(domain)
+	doctype := "io.cozy.files"
+	req := &couchdb.AllDocsRequest{}
+	var results []map[string]interface{}
+	if err := couchdb.GetAllDocs(db, doctype, req, &results); err != nil {
```

Comment:
You can use the [FilesReferencedByView](https://github.com/cozy/cozy-stack/blob/bc8e21249be824144a5feeedf1a3d0786bc817f1/pkg/consts/views.go#L47) to fetch only files with a referenced_by.